README.rst

```diff
@@ -105,11 +105,10 @@ A brief description on how to set-up a Raspberry Pi to use this software.
 
    ::
 
-        $ sudo pip3 install pibooth
+        $ sudo pip3 install pibooth[cups,picamera,gphoto]
```

Comment:
I propose to use more generic args names to be more explicit for noobs:

- `printer` instead of `cups`
- `dslr` instead of `gphoto`
- `??` instead of `picamera` => no idea at the moment, in fact picamera shall not be installed (same for Rpi.GPIO) when pibooth is not installed on raspberry

Reply:
Updated with your names.

For the picamera library, it's installed only if ```platform_machine>="armv0l" and platform_machine<="armv9l"``` is true. So if we try to install it not on a raspberry, it will install nothing.